packages/localize-tools/src/messages.ts

```diff
@@ -169,3 +177,26 @@ export function validateLocalizedPlaceholders(
   }
   return errors;
 }
+
+/**
+ * Given a template string, replace all expression with a provided string (or
+ * "expr" if none provided).
+ *
+ * e.g. `hello ${foo} world ${bar}` -> `hello ${expr} world ${expr}`
+ */
+function replaceExpressionInTemplateString(
+  templateString: string,
+  expression = 'expr'
```

Comment:
Yes, good point. It s unnecessary at the moment. Also renamed to `normalizeExpressionInTemplateString` as well as using string concat rather than array.join().